roles/satellite/tasks/internals/activationkey-hostgroups-deprovision.yml

```diff
@@ -31,42 +30,41 @@
     password: "{{ satellite_pwd }}"
     server_url: "{{ satellite_url }}"
     validate_certs: "{{ satellite_verify_ssl }}"
-    name: "{{ g_prod_activationkey }}"
+    name: "{{ prod_activationkey }}"
     content_view: "{{ deploymentcontentview }}"
-    lifecycle_environment: "{{ g_prod_lifecycleenvironment }}"
+    lifecycle_environment: "{{ prod_lifecycleenvironment }}"
```

Comment:
If you want to be resilient even if they are not defined, you might want to use `prod_lifecycleenvironment | default(omit)`